frameworks/symfony

```diff
@@ -20,14 +24,20 @@ function is_symfony3() {
     test -e "bin/console"
 }
 
+function is_symfony4() {
```

Comment:
There is not much difference between `is_symfony4` and `requires_symfony_flex`. Isn't it possible to factorize a bit?

Reply:
✅